hpat/tests/test_series.py

```diff
@@ -1030,14 +1030,24 @@ def test_impl(S):
         S = pd.Series(['aa', 'bb', np.nan])
         self.assertEqual(hpat_func(S), test_impl(S))
 
-    def test_series_mean1(self):
+    def test_series_mean_default(self):
         def test_impl(S):
             return S.mean()
         hpat_func = hpat.jit(test_impl)
 
         S = pd.Series([np.nan, 2., 3.])
+        self.assertEqual(hpat_func(S), 2.5)
         self.assertEqual(hpat_func(S), test_impl(S))
 
+    def test_series_mean_skipna_false(self):
```

Comment:
Please test both state of the optional value like in https://github.com/IntelPython/hpat/blob/e45cc922afb855f0e5d87e30acc01ef6ef2e4002/hpat/tests/test_series.py#L2097

Reply:
Done